.github/workflows/publish-release.yaml

```diff
@@ -0,0 +1,63 @@
+name: Publishing Files
+on:
+  release:
+    types: [created]
```

Comment:
are we sure this doesn't trigger on creating prerelease too ?

Reply:
Probably, I don't know yet, since this is not yet tested.
https://help.github.com/en/github/automating-your-workflow-with-github-actions/events-that-trigger-workflows#release-event-release